PlexAniSync.py

```diff
@@ -51,11 +51,19 @@ def read_settings(settings_file) -> configparser.ConfigParser:
     return settings
 
 
-if len(sys.argv) > 1:
-    SETTINGS_FILE = sys.argv[1]
-    logger.warning(f"Found settings file parameter and using: {SETTINGS_FILE}")
+# check if setting parameters are set.
+# check if the arg given is empty
+check_settings_arg = os.environ.get("SETTINGS_FILE")
```

Comment:
```suggestion
SETTINGS_FILE = os.getenv("SETTINGS_FILE", "settings.ini")
```

Then almost all the following lines can be removed as only the old `sys.argv[1]` code is needed anymore.

Reply:
> Then almost all the following lines can be removed as only the old `sys.argv[1]` code is needed anymore.
This doesn't seem to work, I tried it but it fails to find `settings.ini` for some reason even tho it calls `settingsupdater.py`